e2/de/messages.txt

```diff
@@ -93,6 +93,7 @@ MISSINGNUMRECRUITS, Anzahl Rekruten fehlt
 MISSINGOFFER, Geldgebot fehlt
 MISSINGPARAMETERS, Parameter STUFE oder REGION fehlt
 MISSINGPASSWORD, Kein Passwort angeben
+MISSINGSTART, ERESSEA partei-nummer "Passwort" fehlt
```

Comment:
Rückgängig gemacht in commit 417a5e98145f6e242dd6dfe952809fe847bd84c4 weil es nicht damit klar kam, dass in der ersten Zeile ein BOM steht.